public/src/app/app.component.html

```diff
@@ -20,6 +20,49 @@
               -->
             </span>
           </span>
+          <span class="navbar-contributions" *ngIf="showContributions">
+            <span class="navbar-contribution" *ngFor="let user of status?.contributions.data.repository.contributions.nodes">
+              <span *ngIf="user.contributions.pullRequests.nodes.length + user.contributions.reviews.nodes.length + user.contributions.issues.nodes.length > 0">
+                <img src="{{user.avatarUrl}}&size=40">
```

Comment:
This is very handy! Instead of hover, can we make these open in a new tab?
For avatars further down the row, the contributions panel ended up scrolling off the page, but I couldn't scroll cause the hover would disappear.

Reply:
Or add the option to click in addition to hover. Since you intended this dashboard to minimize browser tabs :)

Reply:
You just need a bigger screen! For now, I don't want to add more complexity just now -- the click adds interactions that require additional JS... not difficult but perhaps I need to adjust the CSS drop down to keep it on page.

Reply:
fine. I can use my arrow keys to scroll to the right

Reply:
I tweaked this slightly -- it should now be on screen always in most situations.